docs/business-apps/power-automate/get-started/sharepoint-actions-triggers.md

```diff
@@ -233,7 +233,7 @@ Updates the contents of the file specified by the file identifier.
 
 ### Update file properties
 
-Updates the properties stored in columns in a library for the item specified by the item id. To update file contents, use the "Update file" action. When using this with the On-Premises Data Gateway, you may need to manually enter the name of the library to connect to.
+Updates the properties stored in columns in a library for the item specified by the item ID. To update file contents, use the **Update file** action. When using this with the On-Premises Data Gateway, you may need to manually enter the name of the library to connect to.
 
 ### Update item
```

Comment:
Missing description. We can add the following:

_Updates an item in a SharePoint list._

@jackwi-alt  let me know if this is too simple and would want to add more context?